src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,36 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const activeSelectTab = tabs.find(tab => activeTabId === tab.id) || tabs[0];
+
+  const onClick = tab => {
+    if (activeSelectTab.id !== tab.id) {
+      onTabSelected(tab.id);
+    }
+  };
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              className={activeSelectTab.id === tab.id && 'is-active'}
+              data-cy="Tab"
+            >
+              <a
+                data-cy="TabLink"
+                href={`#${tab.id}`}
+                onClick={() => onClick(tab)}
+              >
+                {tab.title}
+              </a>
+            </li>
```

Comment:
You may consider making the tab header a separate component (it is up to you and is not required)